stable/weave-scope/charts/weave-scope-agent/templates/daemonset.yaml

```diff
@@ -0,0 +1,86 @@
+{{- $svcname := "weave-scope" -}}
+{{- $svcport := 80 -}}
+
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  labels:
+    {{- include "weave-scope.helm_std_labels" . | indent 4 }}
+    component: agent
+  name: {{ template "weave-scope.fullname" . }}-agent
+  annotations:
+    {{- include "weave-scope.annotations" . | indent 4 }}
+spec:
+  selector: 
+    matchLabels:
+      app: {{ template "weave-scope.name" . }}
+      release: {{ .Release.Name }}
+      component: agent
+  updateStrategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        {{- include "weave-scope.helm_std_labels" . | indent 8 }}
+        component: agent
+    spec:
+      tolerations:
+        - effect: NoSchedule
+          operator: Exists
+      containers:
+        - name: agent
+          image: {{ required "You must specify the desired container image" .Values.global.image }}:{{ required "You must specify the desired container image tag" .Values.global.imageTag }}
```

Comment:
Image and tag are already specified in `values.yaml`. I would remove the `required` calls.